cra/task-terraform-scan.yaml

```diff
@@ -13,6 +13,8 @@ spec:
     - name: branch
       description: the branch
       default: master
+    - name: commit-id
+      description: git commit id
```

Comment:
Existing user/adopter of this task will be broken as this commit-id parameter is a new expected parameter for this task
So in order to be previous-versio compliant a default value should be defined (asusming "") and take in account by the underlying node program